WordPress/src/main/res/layout/jetpack_login_empty_view.xml

```diff
@@ -2,78 +2,78 @@
 <ScrollView android:layout_width="match_parent"
     android:layout_height="wrap_content"
     xmlns:android="http://schemas.android.com/apk/res/android">
-<androidx.constraintlayout.widget.ConstraintLayout
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    <androidx.constraintlayout.widget.ConstraintLayout
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
 
-    <com.google.android.material.card.MaterialCardView
-        android:id="@+id/user_cardView"
-        style="@style/Widget.EmptyView.CardView.BottomBar"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        app:cardCornerRadius="0dp"
-        app:cardElevation="0dp"
-        tools:visibility="visible"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintBottom_toTopOf="@+id/login_empty_view_illustration"
-        android:visibility="invisible">
+        <com.google.android.material.card.MaterialCardView
+            android:id="@+id/user_cardView"
+            style="@style/Widget.EmptyView.CardView.BottomBar"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            app:cardCornerRadius="0dp"
+            app:cardElevation="0dp"
+            tools:visibility="visible"
```

Comment:
👋 @zwarm !

If I reformat this file the `android:xyz` attributes will go on top of the `tools:xyz`, but this didn't happen here, I wonder what is the problem?

Reply:
@ParaskP7 - Your guess is as good as mine. Constraint layouts seems to be a an AS reformat nightmare. I'm leaving it as it is.

Reply:
@zwarm in my case it was doing at least what I expected with that. Do you want a quick 5' mins pairing to see what is happening on your IDE?

Reply:
@ParaskP7 - Sure, we can compare Editor -> Code Style -> XML in the preferences settings